lib/rules/arrow-parens.js

```diff
@@ -105,6 +105,14 @@ module.exports = {
                 ], `${shouldAddSpaceForAsync ? " " : ""}${paramToken.value}`);
             }
 
+            if (
+                node.params.length === 1 &&
+                (sourceCode.getCommentsBefore(node.params[0]).length > 0 ||
+                sourceCode.getCommentsAfter(node.params[0]).length > 0)
+            ) {
+                return;
+            }
```

Comment:
It looks like this would also wrongly prevent some errors for `"always"`:

```js
/* eslint arrow-parens: ["error", "always"] */

var foo = /**/ a => b;

var bar = a /**/ =>  b;
```

Also, there is an `as-needed` edge case that isn't covered:

```js
/* eslint arrow-parens: ["error", "as-needed"] */

var foo = (a,/**/) => b;
```

Reply:
> It looks like this would also wrongly prevent some errors for `"always"`:
> 
> ```js
> /* eslint arrow-parens: ["error", "always"] */
> 
> var foo = /**/ a => b;
> 
> var bar = a /**/ =>  b;
> ```

let me check them

> ```js
> /* eslint arrow-parens: ["error", "as-needed"] */
> 
> var foo = (a,/**/) => b;
> ```

I will add test case for this. but is will same as without the `,` cause I think the `params` length is still 1 here

Reply:
done